src/Console/Commands/stubs/eloquent/request.stub

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace DummyNamespace;
+
+use CloudCreativity\JsonApi\Http\Requests\RequestHandler;
+
+class Request extends RequestHandler
+{
+
+    /**
+     * @var array
+     */
+    protected $hasOne = [
+        //
+    ];
+
+    /**
+     * @var array
+     */
+    protected $hasMany = [
+        //
+    ];
+}
```

Comment:
For the Eloquent request stub, can we add the following:

``` php
    /**
     * @var array
     */
    protected $allowedFilteringParameters = [
        'id',
    ];
```

Because by default the search supports that as a filtering parameter.